packages/backend/services/db.js

```diff
@@ -1,8 +1,14 @@
-const firebaseImplementation = require("./dbFirebase");
-const localImplementation = require("./dbLocal");
+require("dotenv").config();
 
-// TODO in #47, use one implementation or the other based on .env
-const db = localImplementation;
+const DB_SERVICES = {
+  firebase: "dbFirebase",
+  local: "dbLocal",
+};
+
+const selectedService = process.env.DATABASE_SERVICE ?? "local";
+const dbService = DB_SERVICES[selectedService] ?? DB_SERVICES.local;
+// eslint-disable-next-line import/no-dynamic-require
+const db = require(`./${dbService}`);
```

Comment:
@dgrcode Any alternative to this?

It works, but the linting error threw me off a little bit. From the linting rule:

> The require method from CommonJS is used to import modules from different files. Unlike the ES6 import syntax, it can be given expressions that will be resolved at runtime. While this is sometimes necessary and useful, in most cases it isn't. Using expressions (for instance, concatenating a path and variable) as the argument makes it harder for tools to do static code analysis, or to find where in the codebase a module is used.

Reply:
We can remove the dynamic import with this:
```js
require("dotenv").config();

const selectedService = process.env.DATABASE_SERVICE ?? "local";
const db = selectedService === "local" ? require("./dbLocal") : require("./dbFirebase");
```

If we grow the list of db types we can always do something like:
```js
let db;
switch (selectedService) {
  case ...
    // ... all posibilities

  case "local":
  default:
    db = require("./dbLocal")
}
```

If we were using `import foo from 'foo'` things would be trickier, but with `require()` we can completely avoid running any firebase logic just with the code above.